src/types/api/base.ts

```diff
@@ -29,3 +34,10 @@ export type Location = {
   kladr_code?: string;
   address: string;
 };
+
+export type addWebhooksRequests = {
```

Comment:
request entity could be generic and base for many types of responses, so it should be checked... if it true, it should be named just `Request`